src/Symfony/Component/Ldap/Security/LdapUserProvider.php

```diff
@@ -74,17 +73,10 @@ public function loadUserByUsername(string $username)
 
     public function loadUserByIdentifier(string $identifier): UserInterface
     {
-        try {
-            $this->ldap->bind($this->searchDn, $this->searchPassword);
-            $identifier = $this->ldap->escape($identifier, '', LdapInterface::ESCAPE_FILTER);
-            $query = str_replace(['{username}', '{user_identifier}'], $identifier, $this->defaultSearch);
-            $search = $this->ldap->query($this->baseDn, $query);
-        } catch (ConnectionException $e) {
-            $e = new UserNotFoundException(sprintf('User "%s" not found.', $identifier), 0, $e);
-            $e->setUserIdentifier($identifier);
-
-            throw $e;
-        }
+        $this->ldap->bind($this->searchDn, $this->searchPassword);
```

Comment:
We could opt to catch `InvalidCredentialsException` here in order to throw something more-specific that would indicate it is the search bind credentials (used during enumeration) that are invalid, as opposed to the credentials that the user has provided.